src/native/libs/System.Globalization.Native/pal_collation.m

```diff
@@ -375,4 +375,36 @@ int32_t GlobalizationNative_GetSortKeyNative(const uint16_t* localeName, int32_t
     }
 }
 
+int32_t GlobalizationNative_GetUIUnicodeVersion(void)
```

Comment:
>GlobalizationNative_GetUIUnicodeVersion [](http://example.com/codeflow?start=8&length=39)

This doesn't look right to me. I mean not matching `ucol_getVersion` behavior. 
`ucol_getVersion` can return a different version for different collation. I know you are using LCID in the C# but there are tons of cultures has same LCID (which is 0x1000 as I recall). ObjectC doesn't have API can help returning a better version?

Reply:
If you create a UCollator for "en" and one for "fr", the version values returned by `ucol_getVersion` may be different, because the collation rules (tailoring) for these locales differ. does you implementation will have this behavior?

Reply:
Can you make `GlobalizationNative_GetUIUnicodeVersion` take the locale parameter and return specific version for this locale? Does Objective C has any API that can return the collation version for any locale like ICU does?